qbft/spectest/tests/commit/sorted_decided.go

```diff
@@ -0,0 +1,38 @@
+package commit
+
+import (
+	"crypto/rsa"
+
+	"github.com/ssvlabs/ssv-spec/qbft/spectest/tests"
+	"github.com/ssvlabs/ssv-spec/types"
+	"github.com/ssvlabs/ssv-spec/types/testingutils"
+)
+
+// SortedDecided tests the creation of the decided message that should have sorted signers
+func SortedDecided() tests.SpecTest {
```

Comment:
no output messages?

Reply:
Notice that `ControllerSpecTest.OutputMessages` is unused. For this test, we actually use `ControllerSpecTest.RunInstanceData.ExpectedDecidedState.BroadcastedDecided` to test the broadcasted decided message.

Reply:
Notice that not only does none of the implemented tests define it but the spec test itself doesn't use it.

As I see it, the reason is that the controller spec test wants to test decided values and the decided messages for a list of instances (and not the instance's broadcasted messages).

I think that we should either drop this field or implement it in the tests if wanted.